src/lib/state.ts

```diff
@@ -155,6 +167,14 @@ export const dumpDeviceClass = (
   mandatoryControlledCCs: deviceClass.mandatoryControlledCCs,
 });
 
+export const dumpCommandClass = (
+  commandClass: CommandClass
+): CommandClassState => ({
+  id: commandClass.ccId,
+  name: CommandClasses[commandClass.ccId],
+  version: commandClass.version,
```

Comment:
I guess whether this CC is secure or not is of no interest to HA, but it could be useful for debugging.

Reply:
> I guess whether this CC is secure or not is of no interest to HA, but it could be useful for debugging.

I can add it. How to access it from the CommandClass instance ?

Reply:
nvm, got it